ajax/libs/flipcountdown/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "flipcountdown",
+  "version": "3.0.5",
+  "description": "jQuery Clock,Timer or Counter plugin in retro flip-count down style. The plugin can display the time and number, including floating point.",
+  "npmName": "flipcountdown",
+  "filename": "jquery.flipcountdown.min.js",
```

Comment:
@nolsherry Why your npm auto-update config split by filename? Does'nt you copy from [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md) or other libraries with npm autoupdate? It looks weird.